lib/taxes/taxes-rules-fr.ts

```diff
@@ -654,16 +659,31 @@ export const getFrTaxesForEspp = (
 
   // Process each event
   eventsEspp.forEach((event) => {
+    const hasEtradeCollectedTaxes = isRoughlyEqual(
```

Comment:
can we keep this file etrade agnostic? 
```suggestion
    const hasCollectedTaxes = isRoughlyEqual(
```